core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-content-id.service.spec.ts

```diff
@@ -47,9 +50,22 @@ describe('Translation tab active content id service', function() {
     ttacis = $injector.get('TranslationTabActiveContentIdService');
   }));
 
+  beforeEach(() => {
+    testSubscriptions = new Subscription();
+  });
+
+  afterEach(() => {
+    testSubscriptions.unsubscribe();
+  });
+
+
   it('should correctly set and get active content id', function() {
+    const sampleSubscription = jasmine.createSpy('sampleSubscription');
+    testSubscriptions.add(ttacis.onActiveContentIdChange
+      .subscribe(sampleSubscription));
```

Comment:
Try to break on opening brackets
```suggestion
    testSubscriptions.add(
      ttacis.onActiveContentIdChange.subscribe(
        sampleSubscription));
```

Reply:
Done